packages/auth/amplify_auth_cognito/lib/src/auth_plugin_impl.dart

```diff
@@ -126,6 +120,20 @@ class AmplifyAuthCognito extends AmplifyAuthCognitoDart with AWSDebuggable {
     AmplifyConfig? config,
     required AmplifyAuthProviderRepository authProviderRepo,
   }) async {
+    // Dependencies for AnalyticsMetadataType
+    stateMachine
+      ..addInstance<LegacyNativeDataProvider>(
+        FlutterLegacyNativeDataProvider(),
+      )
+      ..addInstance(
```

Comment:
`addInstance` calls will override previous ones, so calling this here won't do anything since it will be overridden in `super.configure`.

Reply:
Outdated, removed for EndpointInfoStoreManager